ui/perfherder/helpers.js

```diff
@@ -429,38 +429,148 @@ export const addResultsLink = (taskId) => {
   return `${taskLink}${taskId}${resultsPath}`;
 };
 
-const formatAlert = (alert, alertsWithVideos) => {
-  const numFormat = '0,0.00';
-  let amountPct;
+export class Summary {
```

Comment:
This name raises ambiguity concerns. One would get confused about this & an alert summary.